addons/stock_account/models/stock.py

```diff
@@ -75,8 +76,8 @@ class StockMove(models.Model):
     last_done_remaining_qty = fields.Float()
     last_done_qty = fields.Float()
```

Comment:
Will there ever be a relation between stock_move and account_move_line?

https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_move_entries/model/stock_move_entries.py#L51-L52

https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_move_entries/model/stock_move_entries.py#L36-L38

if we want to keep a nitty-gritty detail of accounting valuation this is a must-have.
I know that currently we have a post in the mail_thread in the stock_move/stock_picking
but it is not fully helpful when try to find the right accounting valuation of a product.